src/webextension/firstrun/index.js

```diff
@@ -5,9 +5,9 @@
 import React from "react";
 import ReactDOM from "react-dom";
 
-import Wizard from "./wizard";
+import App from "./wizard";
```

Comment:
Move `wizard.js` to `app.js`? I don't care strongly about this though, and I could always update it when I go through and add tests and suchlike.